lib/publicShare.ts

```diff
@@ -5,6 +5,9 @@
  * @module public
  */
 import { loadState } from '@nextcloud/initial-state'
+import { getBuilder } from '@nextcloud/browser-storage'
+
+const localStorage = getBuilder('shared').persist().build()
```

Comment:
To not shadow global `localStorage` variable and keep a commonly used name.

```suggestion
const browserStorage = getBuilder('shared').persist().build()
```

Reply:
didn't even think of that :sweat_smile: